src/systemverilog/systemverilog.ts

```diff
@@ -354,11 +354,13 @@ export const language = <ILanguage>{
 			// numbers
 			[/\d*\d+[eE]([\-+]?\d+)?(@floatsuffix)/, 'number.float'],
 			[/\d*\.\d+([eE][\-+]?\d+)?(@floatsuffix)/, 'number.float'],
-			[/0[xX][0-9a-fA-F']*[0-9a-fA-F](@integersuffix)/, 'number.hex'],
-			[/0[0-7']*[0-7](@integersuffix)/, 'number.octal'],
-			[/0[bB][0-1']*[0-1](@integersuffix)/, 'number.binary'],
-			[/\d[\d']*\d(@integersuffix)/, 'number'],
-			[/\d(@integersuffix)/, 'number'],
+			[/\d+/, 'number'],
+			[/'[d|D][0-9]*[0-9]/, 'number'],
```

Comment:
You can use [0-9]+ instead of [0-9]*[0-9] here - + is one or more so accomplishes the same thing

Reply:
Similar below